package.json

```diff
@@ -1,17 +1,21 @@
 {
   "name": "eslint-config-sonarqube",
-  "version": "0.6.1",
+  "version": "1.0.0-rc1",
   "description": "ESLint configuration for SonarQube and its plugins.",
   "main": "index.js",
   "license": "LGPL-3.0",
+  "engines": {
+    "node": ">=10.12.0"
+  },
   "peerDependencies": {
-    "eslint-plugin-import": "^2.20.0",
-    "eslint-plugin-jest": "^23.6.0",
-    "eslint-plugin-jsx-a11y": "^6.2.0",
-    "eslint-plugin-promise": "^4.2.0",
-    "eslint-plugin-react": "^7.17.0",
-    "eslint-plugin-react-hooks": "^2.3.0",
-    "eslint-plugin-sonarjs": "^0.5.0",
-    "@typescript-eslint/parser": "*"
+    "@typescript-eslint/eslint-plugin": ">= 4.13.0",
```

Comment:
I added this plugin to make it a bit cleaner to disable rules that we don't want because they are already handled by TS.